build.sbt

```diff
@@ -32,6 +32,11 @@ commands := commands.value.filterNot { command =>
 
 ThisBuild / reproducibleBuildsCheckResolver := Resolver.ApacheMavenStagingRepo
 
+// So that gRPC is properly styled
+ThisBuild / apacheSonatypeArtifactNameProcessor := apacheSonatypeArtifactNameProcessor.value.andThen {
+  _.replaceAll("Grpc", "gRPC")
```

Comment:
Looks great, but better if it match something like GRPC too.

Reply:
the built plugin title cases the name - so Grpc is what is expected - making ths case insensitive is complicated and not worth the hassle